### PR TITLE
Add frontend example for Installation type (B) - package

### PR DESCRIPTION
**Pull request by mansourcodes**
added frontend example for Installation type (B) - package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,16 @@ You can access item children with `$item->children`
...
 
+for Installation type (B), change the namespace to : `Backpack\MenuCRUD\app\Models`
+
+```
+@foreach (Backpack\MenuCRUD\app\Models\MenuItem::getTree(); as $item)
+<a class="no-underline hover:underline p-3" href="{{$item->url()}}">
+    {{ $item->name }}
+</a>
+@endforeach
+```
```
Comment (promatik) [inline README.md:117]:
```suggestion
For Installation type (B), change the namespace to `Backpack\MenuCRUD\app\Models`.
```

**Review by promatik — CHANGES_REQUESTED** [review]
Hi @mansourcodes, good catch, it's better to advise people about this.
I think there is no need to add a new example code block on this, the note is enough, what do you think?

**Comment by mansourcodes** [comment]
Agree 👍

**Review by promatik — APPROVED** [review]

**Review by mansourcodes — COMMENTED** [review]
Great, it's more clear for users to read

**Pull request merged**